### ISSUE TITLE
Should there be a limit on Store client's queries?

### ISSUE THREAD

**Issue by s-tikhomirov** [waku-org/research#36]
A client can overwhelm the server with an excessively long query (an extreme case: "give me all messages form the beginning of times until now"). Should there be a limit on the length of the time frame that a client may request at once?

**Comment by alrevuelta** [comment]
Indeed. Now that I think about it we have it planned in nwaku. https://github.com/waku-org/nwaku/issues/1809
Not just about limiting the time window allowed per request (eg limitted to 1 week) but the bytes sent in general (which takes into account consecutive requests)
